commons/src/main/java/info/metadude/android/eventfahrplan/commons/temporal/Moment.kt

```diff
@@ -149,4 +123,32 @@ data class Moment(var time: Instant) {
      * Returns true if this moment is before given [moment].
      */
     fun isBefore(moment: Moment): Boolean = time.toEpochMilli() < moment.toMilliseconds()
+
+    companion object {
+        /**
+         * Creates a time zone neutral [Moment] instance of current system clock.
+         */
+        @JvmStatic
+        fun now() = Moment(Instant.now())
+
+        /**
+         * Creates a time zone neutral [Moment] instance from given [milliseconds].
+         *
+         * @param milliseconds epoch millis to create instance from
+         */
+        @JvmStatic
+        fun ofEpochMilli(milliseconds: Long) = Moment(Instant.ofEpochMilli(milliseconds))
+
+        /**
+         * Creates a time zone neutral [Moment] instance from given [utcDate].
+         *
+         * @param utcDate must be in ISO-8601 date format, i.e. "yyyy-MM-dd"
+         */
+        fun parseDate(utcDate: String) = Moment(LocalDate.parse(utcDate).atStartOfDay().toInstant(ZoneOffset.UTC))
+
+        /**
+         * Creates a time zone neutral [Moment] instance from the given [ZonedDateTime].
+         */
+        fun ZonedDateTime.toMoment() = Moment(this.withZoneSameInstant(ZoneOffset.UTC).toInstant())
+    }
```

Comment:
It appears a bit odd to have this extension function in the companion object but it looks like we can only have it here.

I would rephrase the comment to:

> Creates a time zone neutral [Moment] instance from this [ZonedDateTime].

Reply:
> I would rephrase the comment to:
> 
> > Creates a time zone neutral [Moment] instance from this [ZonedDateTime].

Done in 6228a4d810e59226ec13a9943249f9e084e2599d.